.claude/statusline-script.sh

```
@@ -0,0 +1,83 @@
+        # Check for staged changes
+        if ! git -C "$current_dir" diff --cached --quiet 2>/dev/null; then
+            status_indicators="${status_indicators}+"
+        fi
+        
+        # Check for untracked files
+        if [ -n "$(git -C "$current_dir" ls-files --others --exclude-standard 2>/dev/null)" ]; then
+            status_indicators="${status_indicators}?"
+        fi
+        
+        # Check ahead/behind status with remote
+        remote_branch=$(git -C "$current_dir" rev-parse --abbrev-ref --symbolic-full-name @{u} 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)